x11-libs/motif/motif-2.3.8-r1.ebuild

```diff
@@ -108,7 +108,7 @@ multilib_src_install_all() {
 	newins "${FILESDIR}"/Mwm.defaults Mwm
 
 	# cleanup
-	rm -rf "${ED}"/usr/share/Xm
+	rm -rf "${ED}"/usr/share/Xm || die
```

Comment:
rm -f won't error out, even if a file isn't found, so the die is pretty much useless here.

Reply:
Updated